tools/downloader/converter.py

```diff
@@ -184,10 +190,13 @@ def convert(context, model):
                 return False
             model_format = 'onnx'
 
+        model_dir = get_model_dir(model)
+
         expanded_mo_args = [
             string.Template(arg).substitute(dl_dir=args.download_dir / model.subdirectory,
                                             mo_dir=mo_path.parent,
-                                            conv_dir=output_dir / model.subdirectory)
+                                            conv_dir=output_dir / model.subdirectory,
+                                            model_dir=model_dir)
```

Comment:
I don't think `model_dir` is the best name for this. First, `dl_dir` and `conv_dir` also pertain to the model. Second, this directory doesn't have the model itself in it.

IMO, it should be named `config_dir`, since it's the directory that contains the model config (and will probably contain the AC config for the model at some point).

Reply:
Done